app/src/main/java/edu/uc/brown5a7/BottomsUp/searchAdaptor/DrinkAdapter.kt

```diff
@@ -0,0 +1,87 @@
+package edu.uc.brown5a7.BottomsUp.searchAdaptor
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Filter
+import android.widget.Toast
+import androidx.recyclerview.widget.RecyclerView
+import edu.uc.brown5a7.BottomsUp.R
+import edu.uc.brown5a7.BottomsUp.dto.Drink
+import kotlinx.android.synthetic.main.row.view.*
+import java.util.*
+import kotlin.collections.ArrayList
+
+class DrinkAdapter(val context: Context, var drinks: ArrayList<Drink>) : RecyclerView.Adapter<DrinkAdapter.MyViewHolder>(){
+    private var drinkListFull: ArrayList<Drink> = ArrayList(drinks)
```

Comment:
Replaced TODO() with ArrayList(drinks).

Changed to variable so that content is changable.